templates/security/email/welcome.html

```diff
@@ -0,0 +1,56 @@
+<style>
+div {
+    margin: auto;
+    width: 50%;
+    background-color: #EAECEE;
+}
+table {
+    margin: auto;
+    text-align: center;
+    font-family: 'Lato', Helvetica, Arial, sans-serif;
+}
+
+.confirmation_button {
+    border-radius: 5px;
+    background-color: #3498DB;
+    border: none;
+    font-size: 15px;
+    text-decoration: none;
+    color: white;
+    padding: 10px 24px;
+}
+</style>
+
+<div>
+    <table>
+        <tr>
+            <td>
+                <a href="{{ url_for('security.login', _external=True) }}">
+                    <img src="{{ url_for('static', filename='images/zenodo-gradient-round.svg', _external=True) }}" class="ui image small"/>
+                </a>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>{{ _('Welcome %(username)s!', username=user.username) }}</p>
+            </td>
+        </tr>
+        {% if security.confirmable %}
+        <tr>
+            <td>
+                <p>{{ _('We are excited to have you get started! </p>
+                <p>First, you need to confirm your account. Just press the button below.') }}</p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                    <a class="btn confirmation_button" href="{{ confirmation_link }}">
+                        {{ _('Confirm my account') }}
+                    </a>
+                </p>
+            </td>
+        </tr>
+        {% endif %}
```

Comment:
what if security.confirmable == False? 
I have a feeling the email will look a bit strange

Reply:
@kpsherva you're right, looks a bit weird: 
<img width="1371" alt="Screenshot 2022-12-05 at 10 55 14" src="https://user-images.githubusercontent.com/61321254/205610296-a88cd3eb-a1d8-46d7-8b5b-63ff783c5fe0.png">

Here is some suggestion: 
<img width="1362" alt="Screenshot 2022-12-05 at 10 59 34" src="https://user-images.githubusercontent.com/61321254/205610434-303a0381-b3e6-49ec-b65c-da18cf731886.png">

Reply:
I like the suggestion! We can improve it in the future if needed :)

Reply:
I think we should implement this suggestion. Maybe in the text we could add:
`Your Zenodo account has been created. We are excited to have you get started!`
So that the message does not look so "cold" and is more informative to the user on why they are receiveing this e-mail.